src/ripple/core/impl/DatabaseCon.cpp

```diff
@@ -46,7 +46,9 @@ DatabaseCon::DatabaseCon (
     {
         try
         {
-            session_ << initStrings[i];
+            soci::statement st = session_.prepare <<
+                initStrings[i];
+            st.execute(true);
```

Comment:
Nit: This works, but why the change? I don't see what's wrong with what was there originally.